### DISCUSSION TITLE
Suggestions for VisiData v2.11.1?

### DISCUSSION THREAD

**Discussion by anjakefala** [saulpw/visidata#1937] [Q&A]
We are preparing a patch release of VisiData [to ship some fixes for Python 3.12](https://github.com/saulpw/visidata/issues/1934). The [branch is ready](https://github.com/saulpw/visidata/pull/1936/commits), but there is still time to add more commits.

Is there anything else you would like? =)

**Comment by LaPingvino** [comment]
Probably for now adding setuptools to the dependencies when packaging (bug #1911) or already fixing that dependency in code (but I think that's not correct for a short term patch release)

**Comment by laktak** [comment]
Since vdsql uses visidata (AFAIK?), could you make it compatible with vdsql again?

At the moment when I run `vdsql postgres://pg:pg@localhost:5432/db` I get:

> TypeError: open_vdsql() got an unexpected keyword argument 'filetype'

**Reply by saulpw** [reply]
Yes, that's next on the list.  It may not make it into the patch release (depends on how big/risky of a fix it is), but should be in the develop branch "soon".

**Reply by laktak** [reply]
I got it running by patching `open_vdsql` and using `vdsql postgresql://...` :)
```
def open_vdsql(vd, p, filetype=None):
```

**Reply by anjakefala** [reply]
Issues tracking the postgres connection via `vdsql` =) :
https://github.com/saulpw/visidata/issues/1854
https://github.com/saulpw/visidata/issues/1743

**Reply by laktak** [reply]
I saw those but both are marked as a duplicate of each other  ;-)

**Comment by laktak** [comment]
Ibis will release v6.0 on Monday. Any chance of supporting it? It adds Oracle backends...

https://github.com/ibis-project/ibis/discussions/6556

**Reply by saulpw** [reply]
We'll be diving into vdsql soon to make it work with newer versions of Ibis.  It won't happen for this patch release though, as the changes aren't already in develop.

**Comment by ssnkhan** [comment]
I'd love to see the green sidebar mode utilised as a view for text-heavy cells: https://github.com/saulpw/visidata/discussions/1912

**Reply by saulpw** [reply]
@ssnkhan yes, that's coming!  But v2.11.1 would be a patch release, and so we'd only want to include existing and low-risk bugfixes.

**Comment by frosencrantz** [comment]
Maybe https://github.com/saulpw/visidata/issues/1621 ?

**Reply by anjakefala** [reply]
I was about to, and then realised that it was already in v2.11!

**Reply by frosencrantz** [reply]
It did not seem that long ago.  Thanks for checking @anjakefala.

**Comment by daviewales** [comment]
Hmm... I'm selfishly eager for some of my PRs to land in a release version!

-  #1717 (#1680)
-  #1730 (JSONL Arrays format)
- #1890
- #1902

**EDIT**: Adding:

- #1949

Not sure how patch-release compatible they are though.

**Reply by saulpw** [reply]
Good checking, @daviewales!  The first two are out of scope, but we'll definitely take the latter two (#1890 and #1902).  I basically scanned the commit messages for `[topic-]`, where the `-` is our notation for "this fixes something that was wrong in the first place".  So you can tag your own PRs similarly in the future if you want us to see it in times like this.

**Reply by daviewales** [reply]
Thanks!
